_posts/2016/2016-04-21-jenadevs-docker-party.md

```diff
@@ -0,0 +1,71 @@
+
+Then the Docker Party, which celebrated the [3rd birthday of Docker](https://www.docker.com/community/docker-birthday-3), was launched! 
+It consisted of an one hour talk including a demo to introduce the technological foundation and nearly three hours of different workshops for every skill level:
+
+  1. **Beginner:** The official Docker birthday app tutorial
+  1. **Advanced:** Building your own Dockerfile accoriding to best practises in conjunction with integration tests 
+  1. **Special:** CoresOS Cluster Workshop
+  1. **Hidden - also known as exploding kittens:** Orchestration Workshop by [Jérôme Petazzoni](https://twitter.com/jpetazzo), the creator of [Docker-in-Docker](https://github.com/jpetazzo/dind) 
+
+## Introduction talk with demo and best practises
+
+In the first place the talk focused on the Docker basics, the components of the Docker ecosystem and the general tooling workflow of building images, running containers and pushing them to the Docker Hub.
+Afterwards the basic commands were demonstrated with some live coding on the terminal, where several containers were also inspected at runtime.
+The second part of the talk considered some best practises for writing your own Dockerfile, which was based on the [official Docker recommendations](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices), tips by [Michael](http://crosbymichael.com/dockerfile-best-practices.html) [Crosby](http://crosbymichael.com/dockerfile-best-practices-take-2.html) as well as the experiences of [epagesdevs](http://twitter.com/epagesdevs) with developing, testing, integrating, deploying and running supportive images for our eCommerce platform.
+This was a good entrypoint to also show some of our typical use cases at ePages, e.g. in the Continuous Delivery Pipeline, for integration testing or in the Operations area for our own and provider infrastructure, which also relies on [Kubeneretes](http://kubernetes.io).
+To finish the talk some typical base images and official app environment images were discussed, too.
+
+## Containers, Containers, Con... err Workshops!
+
+Upfront, all necessary material and code for the workshops was hosted on the jenadevs [GitHub](https://github.com/jenadevs) account which even offers the [complete presentation](https://github.com/jenadevs/jenadevs-meetup-001-docker-party).
+Great news: _All material of **[jenadevs](https://github.com/jenadevs)** is free and will always be open-source!_
+
+### Beginner Workshop
+
+The goal of the beginner workshop was to setup the Docker environment, learn the basic commands, run the workflow and get familiar with Docker Compose.
+For this the participants followed the [official Docker birthday tutorial](https://github.com/docker/docker-birthday-3/blob/master/tutorial.md) while the more experienced Docker users, Benjamin Nothdurft and Kay Abendroth offered assistance.
```

Comment:
The participants followed the..., while the more experienced Docker users offered assistance.